proto/zitadel/idp/v2/idp.proto

```diff
@@ -288,66 +299,83 @@ message SAMLConfig {
+  // using an external account.
   bool is_creation_allowed = 2;
-  // Enable if a new account in ZITADEL should be created automatically when login with an external account.
+  // Enable if a new account in ZITADEL should be created automatically when
+  // login with an external account.
   bool is_auto_creation = 3;
-  // Enable if a the ZITADEL account fields should be updated automatically on each login.
+  // Enable if a the ZITADEL account fields should be updated automatically on
+  // each login.
   bool is_auto_update = 4;
-  // Enable if users should get prompted to link an existing ZITADEL user to an external account if the selected attribute matches.
-  AutoLinkingOption auto_linking = 5 ;
+  // Enable if users should get prompted to link an existing ZITADEL user to an
+  // external account if the selected attribute matches.
+  AutoLinkingOption auto_linking = 5;
 }
 
 enum AutoLinkingOption {
   // AUTO_LINKING_OPTION_UNSPECIFIED disables the auto linking prompt.
   AUTO_LINKING_OPTION_UNSPECIFIED = 0;
-  // AUTO_LINKING_OPTION_USERNAME will use the username of the external user to check for a corresponding ZITADEL user.
+  // AUTO_LINKING_OPTION_USERNAME will use the username of the external user to
+  // check for a corresponding ZITADEL user.
   AUTO_LINKING_OPTION_USERNAME = 1;
-  // AUTO_LINKING_OPTION_EMAIL  will use the email of the external user to check for a corresponding ZITADEL user with the same verified email
-  // Note that in case multiple users match, no prompt will be shown.
+  // AUTO_LINKING_OPTION_EMAIL  will use the email of the external user to check
+  // for a corresponding ZITADEL user with the same verified email Note that in
+  // case multiple users match, no prompt will be shown.
   AUTO_LINKING_OPTION_EMAIL = 2;
 }
 
 message LDAPAttributes {
-  string id_attribute = 1 [(validate.rules).string = {max_len: 200}];
-  string first_name_attribute = 2 [(validate.rules).string = {max_len: 200}];
-  string last_name_attribute = 3 [(validate.rules).string = {max_len: 200}];
-  string display_name_attribute = 4 [(validate.rules).string = {max_len: 200}];
-  string nick_name_attribute = 5 [(validate.rules).string = {max_len: 200}];
-  string preferred_username_attribute = 6 [(validate.rules).string = {max_len: 200}];
-  string email_attribute = 7 [(validate.rules).string = {max_len: 200}];
-  string email_verified_attribute = 8 [(validate.rules).string = {max_len: 200}];
-  string phone_attribute = 9 [(validate.rules).string = {max_len: 200}];
-  string phone_verified_attribute = 10 [(validate.rules).string = {max_len: 200}];
-  string preferred_language_attribute = 11 [(validate.rules).string = {max_len: 200}];
-  string avatar_url_attribute = 12 [(validate.rules).string = {max_len: 200}];
-  string profile_attribute = 13 [(validate.rules).string = {max_len: 200}];
+  string id_attribute = 1 [ (validate.rules).string = {max_len : 200} ];
```

Comment:
Please check this file again for consistent formatting, as well as the certificate attribute to get it returned in get GET and LIST calls.

Reply:
Certificate attribute is still missing in the GET and LIST calls, which means you will never know which certificate you added to the LDAP configuration.